### PR TITLE
KZ-314 Migrate button component

### PR DESCRIPTION
**Pull request by T3ch0o**
## What is this about?

Migrating the existing Buttons.js file from clippings project

## Checklist

- [x] Add buttons
- [x] Add buttons stories
- [x] Add buttons scss

### CODE REVIEW

**Review by EZarkov — CHANGES_REQUESTED** [review]
Remove usage of Icomoon and use internal svg icons

**Review by MartinAndreev — APPROVED** [review]

File: src/assets/scss/button.scss [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    }
+  }
+  &.#{$btn-prefix}--borderless {
+    border: 0;
+  }
+  // social media
+  &.button-twitter,
+  &.button-pinterest {
+    @extend %filled;
+    color: #ffffff;
```
Comment (EZarkov) [inline src/assets/scss/button.scss:196]: This should be a variable.

File: src/assets/scss/button.scss [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  .#{$btn-prefix}--filled {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+}
+
+.#{$btn-prefix}--default {
+  &:hover {
+    svg {
+      fill: #ffffff;
```
Comment (EZarkov) [inline src/assets/scss/button.scss:219]: This should be a variabale.

File: src/core/utils/ClassNameModifierUtil.ts [hunk 3]
```diff
@@ -1,5 +1,9 @@
+import classNames from 'classnames';
+import React from 'react';
+
 const prefix: string = 'p-';
 const formPrefix: string = 'f-';
+const buttonPrefix = 'uni-button'; // TODO: replace with "button" after buttons-refactoring is complete
```
Comment (EZarkov) [inline src/core/utils/ClassNameModifierUtil.ts:6]: This should be renamed to `p-button`

**Review by EZarkov — CHANGES_REQUESTED** [review]

**Review by EZarkov — COMMENTED** [review]
@T3ch0o Do we need button prefix variables at all?

**Comment by T3ch0o** [comment]
@EZarkov Well since all components have some kind of a prefix and especially this one needs variables, according to me we need to keep it that way.

**Review by EZarkov — APPROVED** [review]

**Pull request merged**